src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -159,4 +166,179 @@ public static class PermutationFeatureImportanceExtensions
                 auprc: a.Auprc - b.Auprc);
         }
     }
+
+    public sealed class MetricStatistics
```

Comment:
>MetricStatistics [](start = 24, length = 16)

xml comments for the class missing, especially since it'spublic.  Tell the purpose of the class.

Reply:
I added XML docs to all public classes, public fields, public methods.